packages/codegen/generate.hooks.ts

```diff
@@ -380,64 +380,58 @@ export function generateReactHooks<
     }
 
     function useAllObjects (variables: QueryVariables, options?: QueryHookOptions<IUseObjectsQueryReturnType<GQLObject>, QueryVariables>) {
-        const fetchPolicy = get(options, 'fetchPolicy')
         const skip = get(options, 'skip')
         const { objs, count, error, loading, refetch: _refetch, fetchMore, stopPolling } = useObjects(variables, {
             ...options,
-            fetchPolicy: fetchPolicy && fetchPolicy !== 'no-cache' ? fetchPolicy : 'network-only',
         })
-        const [data, setData] = useState(objs)
-        const [fetchMoreError, setFetchMoreError] = useState()
-        const [firstPageLoaded, setFirstPageLoaded] = useState<boolean>(false)
+        const [fetchMoreError, setFetchMoreError] = useState(null)
 
         // NOTE: returns only the first part of the data
         const refetch: IRefetchType<GQLObject, QueryVariables> = useCallback((...args) => {
-            setData([])
+            setFetchMoreError(null)
             return _refetch(...args)
         }, [_refetch])
 
         useDeepCompareEffect(() => {
-            setData([])
+            setFetchMoreError(null)
         }, [variables])
 
         const loadMore = useCallback(async (skip) => {
             try {
-                const { data: fetchedData } = await fetchMore({
+                await fetchMore({
                     variables: {
                         skip: skip,
                     },
+                    updateQuery (previousData, { fetchMoreResult }) {
+                        // @ts-ignore
+                        const updatedObjs = [...previousData.objs, ...fetchMoreResult.objs]
+                        // @ts-ignore
+                        return { ...previousData, objs: updatedObjs, count: fetchMoreResult.count }
```

Comment:
why tsignore is here? Best to fix it

Reply:
The problem with types.
I'm not sure we'll do anything about it